widgets/data_editor.py

```diff
@@ -827,12 +827,17 @@ def setup_widgets(self):
         self.end = self.add_multiple_decimal_input("End", "end", ["x", "y", "z"],
                                                      -inf, +inf)
 
-        self.unk1 = self.add_integer_input("Unknown", "unk1",
+        self.unk1 = self.add_integer_input("Shortcut Point ID", "unk1",
                                            MIN_UNSIGNED_BYTE, MAX_UNSIGNED_BYTE)
-        self.unk2 = self.add_checkbox("Unknown Flag", "unk2",
-                                      0, 1)
-        self.unk3 = self.add_checkbox("Unknown Flag 2", "unk3",
+        self.unk3 = self.add_checkbox("Double-sided", "unk3",
                                            0, 1)
+        self.unk4 = self.add_checkbox("Lap Checkpoint🧩", "unk4",
+                                           0, 1)
```

Comment:
It seems indentation is not spot on in these two lines. In fact, I think you can fuse both lines into a single line.

(Sames goes for the **Double-sided** parameter a few lines above.)